client/src/components/pages/common/Login/index.js

```diff
@@ -41,127 +32,101 @@ class Login extends React.Component {
+        </div>
+        <div className="right__content">
           <SimpleCard
             ClassName="card_sign"
             content={
-              <>
-                <Typography variant="h4" align="center" color="primary">
-                  Sign In
-                </Typography>
-                <Typography variant="h6" align="center" color="secondary">
-                  Please enter your login details
-                </Typography>
-                <div className="login-input-container">
+              <div>
+                <div className="card_sign__head">
+                  <Typography variant="h6" color="primary">
+                    Sign In
+                  </Typography>
+                  <Typography variant="body2" color="secondary">
+                    Please enter your login details
+                  </Typography>
+                </div>
+                <div className="form">
                   <TextField
+                    className="signForm"
                     id="email"
                     label="Email"
                     variant="outlined"
                     type="email"
                     placeholder="Enter Email"
-                    onChange={this.updateEmail}
-                    isError={message.includes('email must be a valid email')}
+                    onChange={updateEmail}
+                    isError={
+                      message.includes('Please Enter your email') ||
+                      message.includes('Please Check your Email or Password')
+                    }
                     message={
-                      message.includes('email must be a valid email')
+                      message.includes('Please Enter a valid email')
                         ? 'Please Enter a valid email'
                         : ''
                     }
                   />
-                  <Typography variant="body2" color="primary" align="left">
-                    {message.includes('email must be a valid email')
-                      ? 'Please Enter a valid email'
-                      : ''}
-                  </Typography>
                   <TextField
+                    className="signForm"
                     id="password"
                     label="Password"
                     variant="outlined"
                     type="password"
                     placeholder="Enter Password"
-                    onChange={this.updatePassword}
-                    isError={message.includes(
-                      'Must Contain 8 Characters, One Uppercase, One Lowercase, One Number and one special case Character'
-                    )}
+                    onChange={updatePassword}
+                    isError={
+                      message.includes('Please Check your Email or Password') ||
+                      message.includes('Please Enter your password') ||
+                      message.includes('Must Contain 8 Characters')
+                    }
```

Comment:
error message  for password should contain's password pattern 
"Must Contain 8 Characters, One Uppercase, One Lowercase, One Number and one special case Character"